packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -298,8 +300,33 @@ const withChunking = (config) => {
     }
 }
 
+// TODO: Once we create a new project for extensibility we'll have the opportunity to better move this utility
+// to a place that can be reused. This util is also used in the `extensions-loader`.
+const normalizeExtensionsList = (extensions = []) =>
+    extensions.map((extension) => {
+        return {
+            packageName: Array.isArray(extension) ? extension[0] : extension,
+            config: Array.isArray(extension) ? {enabled: true, ...extension[1]} : {enabled: true}
+        }
+    })
+
 const staticFolderCopyPlugin = new CopyPlugin({
-    patterns: [{from: 'app/static/', to: 'static/'}]
+    patterns: [
+        {
+            from: 'app/static/',
+            to: 'static/'
+        },
+        ...normalizeExtensionsList(appConfig?.extensions).map((extension) => {
+            const {packageName} = extension
+            // Parse the extension name out.
+            return {
+                from: `${projectDir}/node_modules/${packageName}/static`,
+                to: `static/${EXTENIONS_NAMESPACE}/${packageName}`,
```

Comment:
is this where all compiled JS and assets live? In my previous work for the server module, i also made a structure inside `/build`, i forgot what the structure looks like but we just want to align so it doesn't end up in multiple ways to store artifacts that belong to an extension...

If time is running short for the sprint, let's create a ticket?